frontend/src/scenes/groups/Group.tsx

```diff
@@ -95,6 +96,15 @@ export function Group(): JSX.Element {
                 >
                     <RelatedGroups id={groupKey} groupTypeIndex={groupTypeIndex} />
                 </TabPane>
+                <TabPane
+                    tab={<span data-attr="groups-related-flags-tab">Feature flags</span>}
+                    key={PersonsTabType.FEATURE_FLAGS}
+                >
+                    <RelatedFeatureFlags
+                        distinctId={groupData.group_key}
+                        groups={{ [groupTypeName]: groupDisplayId(groupKey, groupData.group_properties) }}
```

Comment:
@neilkakkar I referred to the tests but do you mind double checking if I'm passing in the groups correctly? 😬